### PR TITLE
feat(shub_compatibility): send finding to filter regions and change checkType to list

### PR DESCRIPTION
**Pull request by MrCloudSec**
### Context 

When checking global services we need to tell Prowler where to send the findings.
Also, SecurityHub change the RelatedRequirements attribute to a list.

### Description

- When using -f, send SecurityHub finding of global services to the first filter region.
- Change checkType attribute of metadata to list.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: lib/check/models.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ class Check_Metadata_Model(BaseModel):
     # CheckName: str
     CheckTitle: str
     # CheckAlias: str
-    CheckType: str
```
Comment (jfagoagas) [inline lib/check/models.py:60]: 🔝

File: providers/aws/aws_provider.py [hunk 2]
```diff
@@ -322,3 +322,10 @@ def generate_regional_clients(service, audit_info):
         regional_clients[region] = regional_client
         # regional_clients.append(regional_client)
     return regional_clients
+
+
+def get_region_global_service(audit_info):
+    # Check if global service to send the finding to first audited region
+    if audit_info.audited_regions:
+        return audit_info.audited_regions[0]
+    return audit_info.profile_region
```
Comment (jfagoagas) [inline providers/aws/aws_provider.py]: Please, include a test for this function

File: providers/aws/aws_provider.py [hunk 3]
```diff
@@ -322,3 +322,10 @@ def generate_regional_clients(service, audit_info):
         regional_clients[region] = regional_client
         # regional_clients.append(regional_client)
     return regional_clients
+
+
+def get_region_global_service(audit_info):
```
Comment (jfagoagas) [inline providers/aws/aws_provider.py]: Also, include the return type `-> <type>`

**Review by jfagoagas — CHANGES_REQUESTED** [review]
Good job @sergargar, take a look at my comments above, thanks!

File: providers/aws/aws_provider.py (hunk 3 above)
Comment (MrCloudSec) [reply providers/aws/aws_provider.py]: Done

File: providers/aws/aws_provider.py (hunk 2 above)
Comment (MrCloudSec) [reply providers/aws/aws_provider.py]: Why is it necessary for this simple function?

File: providers/aws/aws_provider_test.py [hunk 4]
```diff
@@ -166,3 +167,25 @@ def test_organizations(self):
...
+            credentials=None,
+            assumed_role_info=None,
+            audited_regions=["eu-west-2", "eu-west-1"],
+            organizations_metadata=None,
+        )
+
+        assert (
+            get_region_global_service(input_audit_info)
+            == input_audit_info.audited_regions[0]
+        )
```
Comment (jfagoagas) [inline providers/aws/aws_provider_test.py:191]: Wel done!! 👏

**Review by jfagoagas — APPROVED** [review]
Good job @sergargar !

**Pull request merged**